integration_tests/fixtures/applicationData.json

```diff
@@ -13,6 +13,11 @@
       "consentDate-day": "1"
     }
   },
+  "referrer-details": {
+    "confirm-details": { "name": "Eric Dier", "email": "eric.dier@moj.gov.uk" },
```

Comment:
I don't think we are saving the name/email on the application document? Looks like from CYA page it's just the job title and contact number?

Reply:
aha, good spot! I've added them in as hidden fields in the Nunjucks template.

Reply:
oh no, sorry it's right that we aren't saving them! We don't want two sources of truth...  I think?